lib/log.js

```diff
@@ -70,12 +72,14 @@ Lug.prototype.begin = function (op, request) {
   this.logger.debug(op)
 }
 
-Lug.prototype.event = function (name, data) {
+Lug.prototype.event = function (name, request, data) {
   var e = {
     event: name,
     data: unbuffer(data)
   }
-  process.stdout.write(JSON.stringify(e) + '\n')
+  data.metricsContext = metricsContext.add({},
```

Comment:
I think this should be `e.data.metricsContext`.

`unbuffer` returns a clone of `data` unless you pass the `inplace` argument, so you're setting `metricsContext` on the source object here, not the target.